imports/plugins/core/ui/client/components/table/sortableTable.js

```diff
@@ -0,0 +1,292 @@
+    const { collection, matchingResultsCount, publication } = this.props;
+
+    // Get a count of the number of items matching the current filter.
+    // If no filter is set it will return the total number of items in the collection.
+    const matchingResults = Counts.get(matchingResultsCount);
+
+    const options = {};
+    let skip;
+
+    const pubHandle = Meteor.subscribe(publication, this.state.query, _.extend({
+      skip: skip
+    }, options));
+
+    // optional transform of collection for grid results
+    let results = collection.find(this.state.query, options).fetch();
+    if (this.props.transform) {
+      results = this.props.transform(results);
+    }
+
+    return {
+      loading: !pubHandle.ready(),
+      results: results,
+      matchingResults: matchingResults
+    };
+  }
+
+
+  /**
+   * customFilter() - Replace default filter with customized filter
+   * custom filter is case insensitive
+   * custom filter searches entire string, not just from string start
+   * @param {Object} filter user-typed data
+   * @param {Object} row row info for associated filter
+   * @returns {String} replacement filter
+   */
+  customFilter = (filter, row) => {
+    const id = filter.pivotId || filter.id;
+    if (row[id] !== null && typeof row[id] === "string") {
+      return (row[id] !== undefined
+        ? String(row[id].toLowerCase()).includes(filter.value.toLowerCase())
+        : true);
+    }
+  }
+
+
+  /**
+   * handleFilterInput() - Update state when filter is changed
+   * @param {script} event onChange event when typing in filter field
+   * @param {string} value text field input
+   * @param {string} field input field name to watch
+   * @return {funcion} state for field value
+   */
+  handleFilterInput = (event, value, field) => {
+    this.setState({
+      [field]: value
+    });
+  }
+
+
+  /**
+   * handleClick() - Handle click on table row
+   * @param {object} rowInfo row data passed in from ReactTable
+   * @return {funcion} return onRowClick function prop, or undefined if not supplied
+   */
+  handleClick(rowInfo) {
+    const { onRowClick } = this.props;
+
+    if (typeof onRowClick === "function") {
+      return (
+        onRowClick({
+          className: "sortable-table-row",
+          props: {
+            data: {
+              _id: rowInfo.original._id,
+              type: rowInfo.original.type
+            }
+          }
+        })
+      );
+    }
+  }
+
+
+  /**
+   * renderColumns() - Absorb columnMetadata information from props, output columns to display
+   * @prop {String} columnMetadata - Object of data field, column header
+   * @returns {Object} data filed (string), translated header (string), and minWidth (number / undefined)
+   */
+  renderColumns() {
+    const { columnMetadata } = this.props;
+
+    // Add minWidth = undefined to override 100px default set by ReactTable
+    const displayColumns = columnMetadata.map((element) => {
+      return _.extend({}, element, {
```

Comment:
There is... it's an alias for `assignIn`. I left it as extend because that's what was used in our old Griddle wrapper, I was trying to change as little as possible, but I guess the wrapper is mostly new, so as long as the non-wrapper files didn't change I can blow this up. Will change to `assignIn`